### PR TITLE
New database column, last edited

### PR DESCRIPTION
**Pull request by dpattermann-usgs**

### CODE REVIEW

File: iplover-app/www/js/data.js [hunk 1]
```diff
@@ -259,7 +259,41 @@ iplover.data = (function(){
...
+	
+	//If called, it forces sync when the synchronize page is next loaded.
+	var autoSyncOn = function(){
+		localStorage.syncImmediately = true;
+		return true;
+	};
+	
+	//If the first time called, returns now. 
+	//Otherwise, returns the time set in delaySync.
+//	var timeSincePopup = function(){
```
Comment (thongsav-usgs) [inline iplover-app/www/js/data.js]: Is this commented out for a reason? Also prefer to just delete commented out code.

File: iplover-app/www/js/pages/edit_view_nest_site.js [hunk 2]
```diff
@@ -115,6 +115,11 @@ var save_form = function(e){
     
     //only if set to edited
     rec.changes_synced = false;
+	
+	//Time of last Edit. One is for server, one is for calculations
```
Comment (thongsav-usgs) [inline iplover-app/www/js/pages/edit_view_nest_site.js]: indentation differences, looks like existing code is spaces, not tabs. Would switch to spaces.

File: iplover-app/www/js/pages/home.js [hunk 3]
```diff
@@ -1,14 +1,62 @@
...
+
+function syncPopUp() {
+	var tmp = iplover.data.getChangedRecords();
+	
+	//If there are no changed records, have an empty array. Most other 
+	//problems return false too, so this helps catch various errors.
+	if(!(tmp.length > 0)){return;}
+	var entry = tmp[0];
+	alert("lastEdited " + entry.last_edited);
+//	alert("timestamp " + entry.location_timestamp);
```
Comment (thongsav-usgs) [inline iplover-app/www/js/pages/home.js]: Same question with commented out code.

File: iplover-services/src/main/liquibase/db.1.xml [hunk 4]
```diff
@@ -39,5 +39,11 @@
             <column name="expires" type="date" />
         </createTable>
     </changeSet>
+	<changeSet id="2" author="dpattermann">
```
Comment (thongsav-usgs) [inline iplover-services/src/main/liquibase/db.1.xml]: indentation

File: iplover-app/www/js/pages/home.js (hunk 3 above)
Comment (dpattermann-usgs) [inline iplover-app/www/js/pages/home.js]: Yeah, I messed up the branching on this. It'll be fixed on the next commit.

File: iplover-app/www/js/pages/edit_view_nest_site.js (hunk 2 above)
Comment (dpattermann-usgs) [inline iplover-app/www/js/pages/edit_view_nest_site.js]: Tabs > spaces. But yeah, I should switch to spaces for this project I guess.

**Pull request merged**